build.gradle

```diff
@@ -33,7 +33,7 @@ lombok {
 }
 
 group 'io.odpf'
-version '0.7.0'
+version '0.7.1-beta.1'
```

Comment:
@sumitaich1998 ideally we avoid releasing beta for patch versions. patch can be released directly. we should only use beta and others for a minor and major versions.

Reply:
cool, fixed it , thanks @ravisuhag